Resources/Prototypes/Roles/Jobs/Civ14/TDM/german.yml

```
@@ -0,0 +1,254 @@
+  id: GermanRiflemanGear2
+  equipment:
+    shoes: civ13_shoes_Soviet_sapogi_boots
+    jumpsuit: civ13_uniform_German_camo_SS_uniform
+    head: civ13_head_German_SS_helmet
+# inhand:
+  #  - civ13_boltaction_Gewehr_71
+
+# Machinegunner
+
+- type: job
+  id: GermanMachinegunner 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)